eng/Directory.Build.Data.targets

```diff
@@ -43,6 +45,11 @@
     <None Include="$(RepoEngPath)/images/pkgicon.png" Pack="true" PackagePath=""/>
   </ItemGroup>
 
+  <!-- Don't compile API listing files -->
+  <ItemGroup Condition="'$(GenerateAPIListing)' == 'true'">
+    <Compile Remove="api/*.cs" />
```

Comment:
Does this actually match anything? Below, "api" is sibling to "src" and "test", so why would it be a potential Compile item anyway?

Reply:
Ha, good point, I thought I've put the generated file inside the src directory but now I kind of like having it top level.